WRQ-3732: Fixed Scroller with editable prop to complete editing when focus is leaving by 5-way key on pointer mode (#1522)
enactjs/sandstone: CHANGELOG.md

The following is a curated list of changes in the Enact sandstone module, newest changes on the top.

## [unreleased]

### Fixed

> - `sandstone/Scroller` with `editable` to complete editing when focus is leaving by 5-way key on pointer mode

Reviewer: I'm not sure but how about this suggestion?
```suggestion
- `sandstone/Scroller` with `editable` to complete editing when focus left by 5-way key in pointer mode
```
